### ISSUE TITLE
Question about the security of display.name property of a txn request

### ISSUE THREAD

**Issue by tplooker** [bspk/oauth.xyz-site#14]
Because this is a self attested attribute from the RC thats unverifiable, is there the potential for this property to be abused, i.e through some forms of spoofing attack from malicious RC's?

**Comment by jricher** [comment]
Yes, this is susceptible to the same kinds of impersonations that OAuth's dynamic client registration is. and that needs to be under the security considerations discussion.
